src/libertem/web/shutdown.py

```diff
@@ -0,0 +1,23 @@
+import tornado
+import logging
+
+from .state import SharedState
+
+log = logging.getLogger(__name__)
+
+
+class ShutdownHandler(tornado.web.RequestHandler):
```

Comment:
`ShutdownHandler` looks good now :+1:

Reply:
Btw. it would be good to have test coverage for `ShutdownHandler`, just to make sure the code keeps working across other changes, for example in tornado, or if the structure of the state changes - see https://github.com/LiberTEM/LiberTEM/tree/master/tests/server for some tests that exercise the HTTP API.